stable/fairwinds-insights/README.md

```diff
@@ -130,6 +130,7 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | postgresql.persistence.enabled | bool | `true` | Create Persistent Volume with Postgres |
 | postgresql.replication.enabled | bool | `false` | Replicate Postgres data |
 | postgresql.resources | object | `{"limits":{"cpu":1,"memory":"1Gi"},"requests":{"cpu":"75m","memory":"256Mi"}}` | Resources section for Postgres |
+| encryption.aes.cypherKey | string | `nil` |  |
```

Comment:
```suggestion
| encryption.aes.cypherKey | string | `nil` | Key for encrypting sensitive fields in the database. We recommend adding this to the secret instead of setting it here. |
```